packages/common/src/initiatives/HubInitiative.ts

```diff
@@ -135,14 +135,14 @@ export class HubInitiative
    * @param type
    * @param options Optional hash of Element component options
    */
-  static async getEditorConfig(
-    i18nScope: string,
-    type: EditorConfigType,
-    options: SchemaElementOptions[] = []
-  ): Promise<IEditorConfig> {
-    // Delegate to module fn
-    throw new Error("getEditorConfig Not Implemented for Initiatives");
-  }
+  // static async getEditorConfig(
```

Comment:
Commented out for now... once we're happy w/ how this works for projects, we can author the schemas for initiative